modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItSqlQueriesSystemViewTest.java

```diff
@@ -220,7 +220,7 @@ public void multiStatementWithTransaction() throws InterruptedException {
         long timeAfter = clockService.now().getPhysical();
 
         // "DDL" and "EXPLAIN" queries close cursor automatically.
-        waitForCondition(() -> queryProcessor().runningQueries() == 4, 5_000);
+        assertTrue(waitForCondition(() -> queryProcessor().runningQueries() == 4, 5_000));
```

Comment:
According to the author's idea :sunglasses: , an error should be displayed, where the expected and actual value will be indicated (see the next line of code). But after this change, there will simply be an error without such information, so I would advise you to roll back this change.

Reply:
done